File: 백준/Bronze/17548. Greetings！/Greetings！.cc
#include <iostream>
#include <string>
using namespace std;

int main(){
    string s;
    cin >> s;

    int e_count = count(s.begin(),s.end(),'e');

    string result = "h" + string (e_count *2,'e')  + 'y';
    cout << result << endl;
    return 0;

}

File: 백준/Bronze/17548. Greetings！/Greetings！.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());

    let mut input = reader.lines();

    if let Some(Ok(line)) = input.next() {
        let s = line.trim();
        let fiter = s.chars().filter(|x| *x == 'e').count();
        let result = format!("h{}y", "e".repeat(fiter * 2));
        writeln!(writer, "{}", result).unwrap();
    }
    writer.flush().unwrap();
}
